app/src/main/res/layout/activity_main_2.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:paddingLeft="10dp"
```

Comment:
hard code. Pls define 10dp into "dimens.xml" with name "dp_10" and use it in here!